mlir/lib/Conversion/TosaToRock/TosaToRock.cpp

```diff
@@ -207,35 +207,19 @@ static Value expandTensor(PatternRewriter &rw, Operation *op, Value operand,
   return rw.create<rock::TransformOp>(loc, operand, transform.get());
 }
 
-static std::tuple<StringAttr, std::optional<uint32_t>, rock::GemmFeatures>
-getArchAttributes(Operation *op, Type inputType) {
-  auto func = op->getParentOfType<func::FuncOp>();
-  // auto mod = func->getParentOfType<ModuleOp>();
-
-  // TODO(sjw): get these from options
+static rock::GemmFeatures getTosaOpFeatures(Operation *op, Type inputType) {
```

Comment:
```suggestion
static rock::GemmFeatures getRockGemmOpFeatures(Operation *op, Type inputType) {
```
These features are being applied on Rock ops not tosa op.

Reply:
`getArchAttributes()` , that this new function is replacing, was being passed a Tosa op to calculate the feature. Does it make it confusing to developers to call `getRockGemmOpFeatures()` on Tosa ops? Would something more generic like `getGemmFeaturesFromOp()` be a better name?

Reply:
Yeah `getGemmFeaturesFromOp()` is a better name